components/ingest-service/pipeline/publisher/node_manager_publisher.go

```diff
@@ -79,6 +79,7 @@ func gatherInfoForNode(node backend.Node) (*manager.NodeMetadata, error) {
 		LastContact:     timestamp,
 		SourceId:        node.Ec2.InstanceId,
 		SourceRegion:    node.Ec2.PlacementAvailabilityZone,
```

Comment:
Does node manager use placement availability zone or region? region would be like 'us-west-2' and placement availability zone is like 'us-west-2a'. Region is also available in the ohai data.

Reply:
it uses region -- yes it would be awesome to grab the region instead - i didn't know it was available